lib/main.dart

```diff
@@ -64,6 +60,9 @@ Future<void> main() async {
         store: store,
         child: SentryMobile(),
       ));
+    },
+    (Object error, StackTrace stackTrace) {
+      Sentry.captureException(error, stackTrace: stackTrace);
```

Comment:
those events won't have a mechanism anymore, I'd rather mimic this behavior https://github.com/getsentry/sentry-dart/blob/main/dart/lib/src/default_integrations.dart#L22-L30
mechanism is also used for marking sessions properly (errored/crashed)